contracts/LockRelease.sol

```diff
@@ -105,30 +166,55 @@ contract LockRelease is ILockRelease {
         return (schedule.total * (block.timestamp - start)) / duration;
     }
 
-    /** Returns the total tokens that can be released now. */
+    /** Returns the total tokens that can be released now.
+     *
+     * @param _beneficiary address of the beneficiary
+     * @return uint256 the total tokens that can be released now
+     */
     function getReleasable(address _beneficiary) public view returns (uint256) {
         return getTotalMatured(_beneficiary) - getReleased(_beneficiary);
     }
 
-    /** Returns the total tokens yet to be released to the beneficiary over the total duration. */
+    /** Returns the total tokens yet to be released to the beneficiary over the total duration.
+     *
+     * @param _beneficiary address of the beneficiary
+     * @return uint256 total tokens yet to be released to the beneficiary
+     */
     function getPending(address _beneficiary) external view returns (uint256) {
```

Comment:
@tbwebb22 minor point but maybe we should reorder the functions according to Solidity standards, I think you raised that issue with the fractal contracts before

Reply:
good call, just re-ordered them